### PR TITLE
Upgrade ember-cli-babel to v7

### PR DESCRIPTION
**Pull request by nlfurniss**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "node-test": "mocha node-tests/**/*-test.js"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": ">= 10.*"
```
Comment (YoranBrondsema) [inline package.json:14]: Any particular reason why the minimum Node version must be increased?

Comment (nlfurniss) [reply package.json:14]: This version of Babel doesn’t work with 6, and 10 is the latest version of nice still supported

Comment (YoranBrondsema) [reply package.json:14]: OK!

**Comment by YoranBrondsema** [comment]
I'll publish a new version this weekend, if that's OK with you.

**Comment by YoranBrondsema** [comment]
1.6.0 published

**Pull request merged**